R/print.R

```diff
@@ -39,7 +40,9 @@ print.psis_loo <- function(x, digits = 1, plot_k = FALSE, ...) {
   print.loo(x, digits = digits, ...)
   cat("------\n")
   print_mcse_summary(x, digits = digits)
```

Comment:
Do you prefer printing the r_eff summary above the estimates or should we print it below them where we print the mcse summary? For example, it now looks like this: 

```
Computed from 100 by 50 log-likelihood matrix.
MCSE and ESS estimates assume MCMC draws (r_eff in [0.0, 4.7]).

         Estimate  SE
elpd_loo    -64.3 3.9
p_loo         1.0 0.2
looic       128.5 7.7
------
Monte Carlo SE of elpd_loo is 0.3.
```

but it could be 

```
Computed from 100 by 50 log-likelihood matrix.

         Estimate  SE
elpd_loo    -64.3 3.9
p_loo         1.0 0.2
looic       128.5 7.7
------
MCSE of elpd_loo is 0.3.
MCSE and ESS estimates assume MCMC draws (r_eff in [0.0, 4.7]).
```

In the second one I also changed `Monte Carlo SE of elpd_loo is 0.3.` to `MCSE of elpd_loo is 0.3` to be consistent with `MCSE` and `Monte Carlo SE`.